packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -536,8 +535,7 @@ class Frame extends React.Component {
    */
   role() {
     const { disabled } = this.props;
-    // role="application" needed to allow JAWS to work correctly with the select and use our key event listeners
-    return disabled ? undefined : 'application';
+    return disabled ? undefined : 'combobox';
```

Comment:
If we are removing the role `application` and using `combobox` then we can use it has default role without need of a function

Reply:
Updated. Thanks.